### ISSUE TITLE
Cannot open file in MacOS

### ISSUE THREAD

**Issue by LiveRock** [philenius/flutter_file_picker_desktop#4]
This is the error:
FileSystemException: Cannot retrieve length of file, path = '/Projects/VXX/json.json' (OS Error: No such file or directory, errno = 2)

Same code works well in Windows. Only happen in MacOS.
What did I miss?
Hope to hear from you soon.

**Comment by LiveRock** [comment]
```
final FilePickerResult? result =
                                                                              await pickFiles(
                                                                            allowMultiple: false,
                                                                            allowedExtensions: ['json'],
                                                                            //type: FileType.custom,
                                                                          );
```

The path in 'result.files.single.path' is not correct in MacOS if it is in another volume

**Comment by LiveRock** [comment]
I tried file_picker_desktop: ^1.0.1 on MacOS and couldn't get it work.
Big Sur, Mac Mini M1
The reason is that the path is wrong; it does not take into account the volume.
For example,

/Volumes/NVME-970-EVO-1TB/Projects/file.json

is detected as

/Projects/file.json

How do I resolve this?

**Comment by philenius** [comment]
@LiveRock thank you for letting me know about this bug. It should be fixed with the latest version 1.0.2 which is right now available on pub.dev. Please let me know, if this didn't fix your issue.
